src/chart/pie/PieSeries.js

```diff
@@ -45,6 +48,28 @@ var PieSeries = echarts.extendSeriesModel({
         this._defaultLabelLine(option);
     },
 
+    processOptionSeriesValue: function (option) {
+        var data = option.data;
+
+        for(var i = 0; i < data.length; i++){
+            if(data[i].value instanceof Array){
+                //visualMap component fix me
+                // if(data[i].value[0] instanceof Array){
```

Comment:
What's this commented code about?

Reply:
My answer is incorrect.I am trying to fix it.I want sum array value before compute each Sector ‘s area，please wait for my resubmit.